app/parse.py

```diff
@@ -87,7 +89,8 @@ def get_single_page_products(page_name: str) -> None:
 def get_all_products() -> None:
 
     for page in PAGES_URLS:
-        get_single_page_products(page)
+        with webdriver.Chrome(options=get_options()) as driver:
+            get_single_page_products(page, driver)
```

Comment:
it's better to move driver before for loop because if you scrap 100 pages, it'll create 100 drivers that will take some time. So it's better to create 1 driver and use it to scrap all pages